auth/src/utils/configuredAzureEnv.ts

```diff
@@ -6,42 +6,36 @@
 import * as azureEnv from '@azure/ms-rest-azure-env'; // This package is so small that it's not worth lazy loading
 import * as vscode from 'vscode';
 
-const AzureCloudName = azureEnv.Environment.AzureCloud.name;
-const AzureChinaCloudName = azureEnv.Environment.ChinaCloud.name;
-const AzureUSGovernmentCloudName = azureEnv.Environment.USGovernment.name;
+// These strings come from https://github.com/microsoft/vscode/blob/eac16e9b63a11885b538db3e0b533a02a2fb8143/extensions/microsoft-authentication/package.json#L40-L99
+const CustomCloudConfigurationSection = 'microsoft-sovereign-cloud';
+const CloudEnvironmentSettingName = 'environment';
+const CustomEnvironmentSettingName = 'customEnvironment';
 
-const CloudNameToEndpointSettingValue: { [cloudName: string]: string | undefined } = {};
-CloudNameToEndpointSettingValue[AzureCloudName] = undefined;
-CloudNameToEndpointSettingValue[AzureChinaCloudName] = 'Azure China';
-CloudNameToEndpointSettingValue[AzureUSGovernmentCloudName] = 'Azure US Government';
+const ChinaCloudSettingValue = 'ChinaCloud';
+const USGovernmentSettingValue = 'USGovernment';
+const CustomCloudSettingValue = 'custom';

```

Comment:
Nit: maybe use an enum?

Reply:
FYI those are the property names on the Environment object to get the China & US environments... if it helps